pkgs/development/python-modules/hydra-check/default.nix

```diff
@@ -2,46 +2,39 @@
 , buildPythonPackage
 , pythonOlder
 , fetchFromGitHub
-, docopt
+, poetry-core
 , requests
 , beautifulsoup4
-, mypy
-, types-requests
+, colorama
 }:
 
 buildPythonPackage rec {
   pname = "hydra-check";
-  version = "1.2.0";
-  disabled = pythonOlder "3.5";
+  version = "1.3.4";
+  format = "pyproject";
+
+  disabled = pythonOlder "3.10";
 
   src = fetchFromGitHub {
     owner = "nix-community";
     repo = pname;
-    rev = version;
-    sha256 = "EegoQ8qTrFGFYbCDsbAOE4Afg9haLjYdC0Cux/yvSk8=";
+    rev = "v${version}";
+    sha256 = "sha256-voSbpOPJUPjwzdMLVt2TC/FIi6LKk01PLd/GczOAUR8=";
   };
 
+  nativeBuildInputs = [ poetry-core ];
   propagatedBuildInputs = [
-    docopt
```

Comment:
Still required https://github.com/nix-community/hydra-check/blob/master/pyproject.toml#L39

Reply:
actually that doesn't mean that its required

i pushed a commit that removes that stray reference, no nixpkgs change needed

Reply:
The build failed on my end when switching my config yesterday so I am not sure here.

Reply:
```
       last 10 log lines:
       > Finished executing pipBuildPhase
       > installing
       > Executing pipInstallPhase
       > /build/source/dist /build/source
       > Processing ./hydra_check-1.2.0-py2.py3-none-any.whl
       > Requirement already satisfied: requests in /nix/store/vc3ckbxix6hpyzmvlpkzkjs7w5k39hq4-python3.10-requests-2.27.1/lib/python3.10/site-packages (from hydra-check==1.2.0) (2.27.1)
       > Requirement already satisfied: beautifulsoup4 in /nix/store/kwrym5sg605kn7kfk5nd1qhkh4slv3jq-python3.10-beautifulsoup4-4.10.0/lib/python3.10/site-packages (from hydra-check==1.2.0) (4.10.0)
       > ERROR: Could not find a version that satisfies the requirement docopt (from hydra-check) (from versions: none)
       > ERROR: No matching distribution found for docopt
       >
       For full logs, run 'nix log /nix/store/6g8lgjpch17mm8h10p4jsrsmraidwfl4-python3.10-hydra-check-1.3.4.drv'.
```

Reply:
builds for me on master
also the processing line is `python3.10-hydra-check> Processing ./hydra_check-1.3.4-py3-none-any.whl`

Reply:
Turns out I had some old overlay which I forgot about... Sorry about the noise.